City-of-Helsinki/drupal-helfi-strategia
# UHF-2520: Translate service list correctly

How to test:

1. `make drush-cim`
2. Import tpr services if you don't have them already and publish at least 5.
3. Create a landing page with paragraph "Service list" and add the 5 services.
4. Save the page and make sure the load more button says "Load more services" and result counter says "5 services".
5. Make sure the values are translated to Finnish as well.

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/77
https://github.com/City-of-Helsinki/drupal-helfi/pull/192
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/82
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/157
https://github.com/City-of-Helsinki/drupal-hdbt/pull/146
